backend/src/main/java/fk/prof/backend/aggregator/AggregationStatus.java

```diff
@@ -0,0 +1,11 @@
+package fk.prof.backend.aggregator;
+
+public enum AggregationStatus {
```

Comment:
Yes, should be more descriptive here. Most of this is explained in doc, will add the description here